types/joi/index.d.ts

```diff
@@ -909,6 +909,28 @@ export interface BinarySchema extends AnySchema {
 }
 
 export interface DateSchema extends AnySchema {
+    /**
+     * Specifies that the value must be greater than date.
+     * Notes: 'now' can be passed in lieu of date so as to always compare relatively to the current date,
+     * allowing to explicitly ensure a date is either in the past or in the future.
+     * It can also be a reference to another field.
+     */
+    greater(date: Date): this;
+    greater(date: number): this;
+    greater(date: string): this;
+    greater(date: Reference): this;
+
+    /**
+     * Specifies that the value must be less than date.
+     * Notes: 'now' can be passed in lieu of date so as to always compare relatively to the current date,
+     * allowing to explicitly ensure a date is either in the past or in the future.
+     * It can also be a reference to another field.
+     */
+    less(date: Date): this;
+    less(date: number): this;
+    less(date: string): this;
```

Comment:
Can you please add a specific overload for `'now'`, this make it easier to use.

Reply:
Alternatively you can just merge all of these overloads together, otherwise only the first one will get doc tags.

Reply:
I thought about both of the suggestions, but for the consistency sake kept following the same convention that was used with other methods for that specific definition.